### ISSUE TITLE
code command does not include closeBrowser dependency

### ISSUE THREAD

**Issue by scraymondjr** [getgauge/taiko#315]
### Expected behavior
The code generated from `.code` command should include the `closeBrowser` function as a dependency from `taiko`. The generated code should (generally) be runnable as-is, but currently always have to add the dependency for it to run.

### Actual behavior
The generated code references `closeBrowser` but does not declare it, causing execution of the code to fail for missing dependency.

### Steps to reproduce
1. Open REPL
2. Run `openBrowser()`
3. Run `.code`
4a. Can see the generated code references `closeBrowser` but does not declare it.
4b. Execute code to see error ```ReferenceError: closeBrowser is not defined```

### Versions
```
$ node -v
v11.2.0
$ taiko -v
Version: 0.4.0 (Chromium:72.0.3606.0)
```

**Comment by BugDiver** [comment]
@scraymondjr We could replicate the issue at our end. A fix for this is already in place via e3952e957e4c49f248c26b1fd9ce7505cc8b57a8.
You could try to install it from the source by running `npm install git+https://git@github.com/getgauge/taiko.git`

**Comment by sguptatw** [comment]
Kindly feel free to re-open this issue if you face it after commit e3952e957e4c49f248c26b1fd9ce7505cc8b57a8
